addon/components/-private/base-pop-menu.js

```diff
@@ -251,9 +271,58 @@ export default class BasePopMenuComponent extends Component {
 
     } else if (closest(target, '[data-close]:not([disabled])')) {
       // We still want to allow purposeful closing of the popover from within,
-      // so this closes the popover if the click target has [data-popover-close] attribute
+      // so this closes the popover if the click target has [data-close] attribute
 
       this._removePopover();
     }
   };
+
+  /**
+   * When focus is on the trigger, determine specific key actions
+   */
+  _triggerKeyHandler = (event) => {
+    const keyCode = event.code;
+
+    // Pressing Enter or Space opens the popper
+    if (
+      (keyCode == 'Enter' || keyCode == 'Space')
+      && !this.get('isOpen')
+      ) {
```

Comment:
minor, but formatting seems a little weird here, I think it should be indented less

Reply:
Ok yeah I agree this was weird. linter wouldn't let me put the arguments on 2 lines unless I did it this way. Just gonna put it back on one line again

Reply:
The rule should be that for multiline `if` statements, the closing paren should be the same indentation level as the `if`:

```js
if (
  (keyCode == 'Enter' || keyCode == 'Space')
  && !this.get('isOpen')
) {

}
```

if it wasn't letting you do that then we may need to work on the lint rule